### PR TITLE
[bitnami/discourse] adding extraPorts gives invalid YAML error

### PR DESCRIPTION
**Pull request by JustinAvaLabs**
**Description of the change**
adding extraPorts gives error:
YAML parse error on discourse/templates/service.yaml: error converting YAML to JSON: yaml: line 21: mapping values are not allowed in this context

This change will convert input to yaml without going through the weird bitnami common.tplvalues template.

**Benefits**
Will let me add an https port with http redirect to the bitnami discourse helm chart loadbalancer service

**Possible drawbacks**
I might be wrong

**Applicable issues**
none found

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/discourse/templates/service.yaml [hunk 1]
```diff
@@ -34,6 +34,6 @@ spec:
       nodePort: null
       {{- end }}
     {{- if .Values.service.extraPorts }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | indent 4 }}
+    {{- toYaml .Values.ingress.extraPorts | nindent 4 }}
```
Comment (miguelaeh) [inline bitnami/discourse/templates/service.yaml:37]: `.Values.ingress.extraPorts` does not exist in the values.yaml as a parameter

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Hi @JustinAvaLabs ,
The bitnami common functions are our standard to do thing easier when maintaining the charts so we should not remove them.
Also, please complete the checklist on the PR template in order to ensure you followed all the steps like bumping the chart version.

File: bitnami/discourse/templates/service.yaml (hunk 1 above)
Comment (miguelaeh) [inline bitnami/discourse/templates/service.yaml:37]:
```suggestion
   {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | nindent 4 }}
```

This change should be enought

Comment (JustinAvaLabs) [reply bitnami/discourse/templates/service.yaml:37]: Forgot the switch "ingress" with "service", but the issue still stands. 

I cannot get the service.extraPorts field to template properly using the bitnami.common.tplvalues function.  

I found   "{{- toYaml .Values.ingress.extraPorts | nindent 4 }}" in the same "bitnami/discourse" chart, in the ingress.yml file for a similar field (ingress.extraTLS).  This field actually works though, so I believe switching this change would fix it.

Comment (JustinAvaLabs) [reply bitnami/discourse/templates/service.yaml:37]: Is this solution just to switch indent with nindent?

Comment (JustinAvaLabs) [reply bitnami/discourse/templates/service.yaml:37]: UPdated with bumped chart version

Comment (miguelaeh) [reply bitnami/discourse/templates/service.yaml:37]: Yes changing indent by nindent  works

Comment (miguelaeh) [reply bitnami/discourse/templates/service.yaml:37]: Check my suggestion below, I checked it and seems to work just changing indent by nindent

**Review by miguelaeh — COMMENTED** [review]
Hi @JustinAvaLabs ,
I replied to your comments, the suggestion above should fix the issue, it is matter of changing indent by nindent

**Comment by JustinAvaLabs** [comment]
@miguelaeh  Thanks for the update/review, I have committed your changes.

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**